package.json

```diff
@@ -26,32 +43,33 @@
 		"url": "https://github.com/ComfortablyCoding/strapi-plugin-publisher/issues"
 	},
 	"dependencies": {
-		"@strapi/design-system": "^1.13.1",
-		"@strapi/helper-plugin": "^4.0.0",
-		"@strapi/utils": "^4.0.0",
-		"@strapi/icons": "^1.13.1",
-		"prop-types": "^15.8.1",
+		"@strapi/design-system": "2.0.0-rc.11",
+		"@strapi/icons": "2.0.0-rc.11",
+		"@strapi/utils": "5.2.0",
+		"@strapi/strapi": "^5.2.0",
```

Comment:
These should ideally be peerDependencies. See https://github.com/pluginpal/strapi-plugin-config-sync/blob/master/package.json